cmd/fleet/handleCheckin.go

```diff
@@ -476,10 +478,17 @@ func processPolicy(ctx context.Context, zlog zerolog.Logger, bulker bulk.Bulk, a
 			dl.FieldDefaultApiKeyId:             defaultOutputApiKey.Id,
 			dl.FieldPolicyOutputPermissionsHash: pp.Default.Role.Sha2,
 		}
+		if agent.DefaultApiKeyId != "" {
+			fields[dl.FieldDefaultApiKeyHistory] = model.DefaultApiKeyHistoryItems{
+				ApiKey:    agent.DefaultApiKey,
```

Comment:
Do we need to keep the old key material in the history?

Reply:
> Do we need to keep the old key material in the history?

Not really at the moment. Just added all the default_api_key related properties there. Can remove, let me know.

Reply:
Fewer secrets lying around the better I think.

Reply:
Done.
<img width="433" alt="Screen Shot 2021-11-17 at 10 20 16 AM" src="https://user-images.githubusercontent.com/872351/142229030-348d429f-07ab-4cb8-8b84-08ce4f72c45d.png">

Reply:
![image](https://user-images.githubusercontent.com/872351/142229390-01ccd605-95a0-44d3-8a38-9f04d1ad875b.png)